src/device-registry/routes/v1/sites.js

```diff
@@ -184,43 +184,23 @@ router.post(
         .matches(constants.LATITUDE_REGEX, "i")
         .withMessage("the latitude provided is not valid")
         .bail()
-        .custom((value) => {
-          let dp = decimalPlaces(value);
```

Comment:
Thanks! This is spot on! Thanks @BenjaminSsempala 

Just a quick thought that I will also pass on to @NoahNsimbe , should we totally remove this or just decrease the number of required decimal places?

Reply:
I think it depends on the use case. AirQo devices require a minimum of 5 decimal places for easy device location during maintenance activities.

Other organisations might not require a minimum of 5 decimal places. 

Is it possible to let every organisation configure a minimum number for their devices, 5 decimal places being the recommended/default ???? 

@BenjaminSsempala @Baalmart

Reply:
Sounds like a good idea @NoahNsimbe 

But at the moment, we do not yet properly integrated Organization wide configurations.

I guess the main motivation for these decimal places was accuracy and adherence to the world standards of AQCSV which require even more decimal places.

So this is somehow not just about preference of any organization but it is more about the `quality` of  our GPS data.

Reply:
The more the number coordinates, the more accurate the GPS reading.  @NoahNsimbe / @BenjaminSsempala 
In the meantime:

1. Could we just perhaps use 2 decimal places as we work on organisation wide settings
2. Or just eliminate this input valiidation like how this PR is doing it noa?

Reply:
@Baalmart so should I shift it to 2 decimal places or we eliminate?

Reply:
@BenjaminSsempala , I am still debating the next step with @NoahNsimbe now.

Shall share feedback in a few mins...

Reply:
Hi @BenjaminSsempala , so after a few deliberations with @NoahNsimbe , we have decided to put a short pause on this and workshop around it since there is a lot to consider before we make this change final.

In addition to the existing label, please feel free to add a `work-in-progress` label to this PR as we continue to deliberate (workshop) on the best way forward on this issue.

Thanks again!! This is much much appreciated!

Reply:
Alright then. Thank you


> In addition to the existing label, please feel free to add a `work-in-progress` label to this PR as we continue to deliberate (workshop) on the best way forward on this issue.
> 
> Thanks again!! This is much much appreciated!

Reply:
Hi @BenjaminSsempala , in the short term, let @NoahNsimbe share the GPS coordinates of those devices and we help him create the Sites using approximate coordinates.

In the [endpoint for approximate coordinates](https://airqo-engineering.postman.co/workspace/internal~7a8e9fad-2d13-4d27-af27-f1f4e9a114bf/request/12513372-ceb6c6b9-b0b5-40e3-bd64-47213c8cbea2), one can specify how far (in `kms`) they would like to be from any given GPS coordinates.

1. This will save us from going back and forth and ensure that we do not compromise the majority use-cases of our system. We are attempting to comply with the AQCSV standards as we move towards Platform interoperability. So we need to be mindful when making changes to the system to avoid frustrating ourselves in the near future.
2. As we use that endpoint for approximate coordinates, here is an example where we want to get GPS coordinates which are 5M  (`0.005` KM) from provided GPS coordinates. **See attached screenshot.**
3. In summary, I am not comfortable with us removing these input validations, they will set us back yet we are moving forward. So **AFTER** ensuring that we are all aligned, we shall just delete this PR accordingly.

<img width="918" alt="Screenshot 2023-02-19 at 22 02 49" src="https://user-images.githubusercontent.com/1590213/219969610-03c27d42-653d-4cc4-bac9-b13a28910886.png">

Reply:
Thank you @Baalmart ,
@NoahNsimbe  From the discussion we had today in the physical session, the conclusion was to use approximate coordinates which would meet the input validations, hence the input validations remaining.